userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -22,6 +22,21 @@ The full setup docs can be found [here](/docs/tutorial-basics/setup#prerequisite
 
 ## Simple install from a profile URL
 
+
+### Bootstrapping your local git repository
+When installing profiles you will have to provide the `--git-repository` flag (See [below](#the-git-repository-flag)) unless
+you have bootstrapped your local git repository. To bootstrap your local git repository run the following:
+
+```bash
+pctl bootstrap --git-repository <namespace>/<name> <path-to-git-repo>
```

Comment:
A bit more info about how this is actually referring to flux namespace would be great!
Also what is `<name>` referring to?

Reply:
Also is `<path-to-git-repo>` a relative path? So if you were to use this command in the root of your repo it would be `.`? On second thought, maybe this would be clearer if it were a real-life example with some commented out explanation 🤔

Reply:
> A bit more info about how this is actually referring to flux namespace would be great!
Also what is <name> referring to?

I think this information exists somewhere else on the page? Maybe copy or rearrange?

Reply:
I did point to it, but I've also provided a brief summary again